src/app/components/ViewJSONFormat/ViewJSONFormat.tsx

```diff
@@ -109,7 +109,7 @@ export const ViewJSONFormat: FunctionComponent<ViewJSONFormatProps> = ({
       (key, value) => {
         return dataToHide.indexOf(key) === -1
           ? value
-          : '*'.repeat(value.length);
+          : '*'.repeat(value.length).substring(0, 5);
```

Comment:
Maybe just `'*'.repeat(5)` and a a comment if the intent isn't clear?  No need to allocate a string of whatever length just to slice it down to a known length each time.

Reply:
Thanks @gashcrumb. You are awesome!

Reply:
:blush: